### ISSUE TITLE
Chisel 2 Break EMC values

### ISSUE THREAD

**Issue by Sites94** [Chisel-2/Chisel-2#363]
When updating to the new version of chisel which adds EE3 EMC values, it overwrites Dynamic EMC so for example if i am trying to craft a disk drive from AE2, everything has a preset EMC value, but the final product has a missing EMC value since the dynamic EMC is broken, this does not happen with earlier versions of chisel 2 such as 2.3.10, it is only happening from version 2.4 onwards

**Comment by TheCricket** [comment]
we don't overwrite DynEMC

**Comment by Sites94** [comment]
I'm not blaming you guys dw, i'm just trying to figure out why EMC just dissappears, sorry if you feel like i have blamed you in any way

**Comment by TheCricket** [comment]
Its the wording :P

**Comment by Sites94** [comment]
Yeah, sorry i've been trying to figure this problem out for 2 days straight and its now 1 AM here :P i'll keep looking until i find it, thanks and good job Cricket :)
